src/common/snippets/include/snippets/op/load.hpp

```diff
@@ -20,11 +20,15 @@ namespace op {
  */
 class Load : public MemoryAccess {
 public:
-    OPENVINO_OP("Load", "SnippetsOpset");
+    OPENVINO_OP("Load", "SnippetsOpset", MemoryAccess);
 
     Load(const Output<Node>& x, const size_t count = 1lu, const size_t offset = 0lu);
     Load() = default;
 
+    size_t get_offset() const { return get_input_port_descriptor(0).m_offset; }
+    size_t get_count() const { return get_input_port_descriptor(0).m_count; }
```

Comment:
Minor (here and in LIR loop fusion also) when the class members are public (as for PortDescriptor), there is no need for the `m_` prefix, do it would be simply `get_input_port_descriptor(0).offset`. What do you think?